doc/content/gateways/concepts/mtls-authentication/index.md

```diff
@@ -0,0 +1,84 @@
+---
+title: "mTLS Authentication"
+description: ""
+---
+
+{{% tts %}} supports mutual TLS to authenticate {{% lbs %}} CUPS connections in selected deployments.
+
+<!--more-->
+
+{{% tts %}} does not natively terminate mTLS connections. This is done by a proxy. The proxy is expected to forward the sanitized client certificate chain as an HTTP header. To check if your deployment supports it, contact the operator.
```

Comment:
TTS terminates mTLS just fine: for MQTT and Basic Station there are TLS listeners.